src/sonic-yang-models/tests/yang_model_tests/tests/restapi.json

```diff
@@ -0,0 +1,9 @@
+{
+    "RESTAPI_TABLE_WITH_INCORRECT_CERT": {
+        "desc": "TABLE_WITH_INCORRECT_CERT failure.",
+        "eStrKey": "Pattern"
+    },
+    "RESTAPI_TABLE_WITH_VALID_CONFIG": {
+        "desc": "TABLE WITH VALID CONFIG."
```

Comment:
"RESTAPI TABLE WITH VALID CONFIG."

Desc is helpful in case of test failures. Thx